app/controllers/project_comments_controller.rb

```diff
@@ -72,7 +72,9 @@ def create
                 comment: @comment
               }
             ),
-            date: @comment.created_at.strftime('%d.%m.%Y')
+            date: @comment.created_at.strftime('%d.%m.%Y'),
+            linked_id: @project.id,
+            counter: @project.project_comments.count
```

Comment:
Same here, add comment `# Used for counter badge` or something along those lines.